src/chemistry/Chemistry.hpp

```diff
@@ -7,6 +7,7 @@
 //==============================================================================
 /// \file Chemistry.hpp
 /// \brief Defines methods for integrating primordial chemical network using Microphysics
+/// \Authors: Piyush Sharda and Benjamin Wibking
```

Comment:
Use this syntax for authors: https://www.doxygen.nl/manual/commands.html#cmdauthor